src/sagemaker/workflow/utilities.py

```diff
@@ -408,3 +423,28 @@ def execute_job_functions(step_args: _StepArguments):
     chained_args = step_args.func(*step_args.func_args, **step_args.func_kwargs)
     if isinstance(chained_args, _StepArguments):
         execute_job_functions(chained_args)
+
+
+def strip_timestamp_from_job_name(request_dict, job_key):
```

Comment:
I believe this method does the same thing- can you reuse it? 
https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L128